frontend/src/employee-mobile-frontend/child-attendance/ChildSubListItem.tsx

```diff
@@ -115,7 +115,7 @@ export default React.memo(function ChildSubListItem({
     if (reservationData.outOnBackupPlacement)
       return [i18n.attendances.confirmedDays.inOtherUnit]
 
-    const [withTimes, noTimes] = partition(
+    const [withTimes] = partition(
```

Comment:
if we don't need the other half then `filter` makes more sense

Reply:
Changed to `filter()`!